config/maintainers.json

```diff
@@ -11,16 +11,6 @@
       "avatar_url": null,
       "bio": null
     },
-    {
-      "github_username": "mhinz",
-      "alumnus": false,
```

Comment:
Are you certain you want to _remove_ yourself from this list? We usually give credit to people by switching the `alumnus` to `true`. If you no longer want to show up on the website, switch on `alumnus` and switch off `show_on_website`. Of course it's your right to be removed completely, but you might want to consider this.

Reply:
Git history is credit enough for me, but thanks for the consideration. :)